contributors/contribute-to-a-gerrit-project.md

```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
 
 ## Corporate Contributor
 
-EasyCLA presents a field where you confirm your association to the company.
+EasyCLA presents a review agreement link where you confirm your association with the company.
 
-1. Select the Corporate CLA that you want to sign.
+1. Select **Corporate CLA**.
 
-   An agreement field appears:
+   ![Gerrit Corporate CLA Agreement](/.gitbook/assets/corporate-cla.png)
+
+2. Click the **Please review the agreement link** and then click the message link that appears:
+
+   ![Gerrit Sign ICLA Link](../.gitbook/assets/cla-gerrit-icla-proceed-to-sign-cla.png)
+   
+3. Sign in to EasyCLA if you are prompted.
+   
+4. Select **Company**.
+
+   To contribute to this project, you must be authorized under a signed Contributor License Agreement. You are contributing on behalf of    your work for a company.
```

Comment:
Not sure about this big space here between "behalf of" and "your".

Reply:
deleted the extra tab space.